examples/bigquery-snowflake-tables-migration-utility/BQ_Table_Creator/bq_table_creator.py

```diff
@@ -0,0 +1,418 @@
+# Copyright 2022 Google LLC
+
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+
+#        http://www.apache.org/licenses/LICENSE-2.0
+
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+"""Module to create bq table from the extracted ddl"""
+
+
+import sys
+import argparse
+import json
+import ast
+import datetime
+from google.cloud import bigquery
+from google.api_core.exceptions import NotFound
+from google.cloud import storage
+
+
+def create_log_table(project_id, target_dataset, client):
+    """
+    Function to create log table
+    """
+    table_id = f"{project_id}.{target_dataset}.report_status_log_tbl"
+    schema = [
+        bigquery.SchemaField("Timestamp", "STRING"),
+        bigquery.SchemaField("FilePath", "STRING"),
+        bigquery.SchemaField("Schemaname", "STRING"),
+        bigquery.SchemaField("TableName", "STRING"),
+        bigquery.SchemaField("Category", "STRING"),
+        bigquery.SchemaField("Message", "STRING"),
+        bigquery.SchemaField("Status", "STRING"),
+        bigquery.SchemaField("Action", "STRING"),
+    ]
+    table_address = bigquery.Table(table_id, schema=schema)
+    table_ref = client.create_table(table_address, exists_ok=True)
+    return table_ref
+
+
+def log_table_data(table, client, records):
```

Comment:
It seems to me that the same function is defined in `snowflake_ddl_extraction.py`. Could this function be extracted to a common py file to be leveraged by both script without duplicating code?